stm/vernac_classifier.ml

```diff
@@ -95,36 +95,26 @@ let classify_vernac e =
     | VernacSetOption (_, ["Default";"Proof";"Using"],_) -> VtSideff [], VtNow
     (* StartProof *)
     | VernacDefinition ((Decl_kinds.DoDischarge,_),({v=i},_),ProveBody _) ->
-      VtStartProof(Doesn'tGuaranteeOpacity, idents_of_name i), VtLater
+      VtStartProof (idents_of_name i), VtLater
 
     | VernacDefinition (_,({v=i},_),ProveBody _) ->
-       let guarantee = if poly then Doesn'tGuaranteeOpacity else GuaranteesOpacity in
```

Comment:
How do such definitions now work?

Reply:
I guess that was used a hack to indicate "don't delegate poly proofs`. If so, that should be indeed accounted in the main classification path right?

Reply:
This is the a part of the system that I (sadly) have no real idea, how delegation an universes work.

Reply:
Do we have any tests that we do the right thing with polymorphic delegation?